provision/testdata/with_refreshtime.apic.txt

```diff
@@ -0,0 +1,916 @@
+/api/mo/uni/infra/vlanns-[kube-pool]-static.json
```

Comment:
I believe there is no refresh time being sent in anywhere in this apic configuration. As such, can we not use an existing output file for validation? If so, I would prefer doing that as opposed to creating a new one. Else for someone not familiar, it gives the impression that there is something more specific to the refresh time that you are trying to validate here and hence creating the new file (since the whole file is new, there is no diff with a previous version).